src/fabric/src/fabric2_active_tasks.erl

```diff
@@ -0,0 +1,45 @@
+-module(fabric2_active_tasks).
+
+
+-export([
+    get_active_tasks/0,
+    register_tasks/1
+]).
+
+
+-include_lib("couch/include/couch_db.hrl").
+-include("fabric2.hrl").
+
+
+-define(ACTIVE_TASK_JOB_TYPES, [<<"views">>, <<"replication">>]).
+
+
+register_tasks(Mod) when is_atom(Mod) ->
+    ActiveTasks = lists:usort([Mod | registrations()]),
+    application:set_env(fabric, active_tasks, ActiveTasks).
+
+
+get_active_tasks() ->
+    ActiveTasks = lists:foldl(fun(Type, TaskAcc) ->
+        Tasks = couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(undefined),
+            fun(JTx) ->
+                JobIds = couch_jobs:get_active_jobs(JTx, Type),
+                maps:map(fun(JobId, _) ->
+                    {ok, Data} = couch_jobs:get_job_data(JTx, Type, JobId),
+                    populate_active_tasks(Type, Data)
```

Comment:
For consistency wonder if we could pass the JobId in as well, so it might be `populate_active_tasks(Type, JobId, JobData)`